dashboard/db/migrate/20160913212311_add_unique_constraint_to_professional_learning_partners.rb

```diff
@@ -0,0 +1,8 @@
+class AddUniqueConstraintToProfessionalLearningPartners < ActiveRecord::Migration[5.0]
+  def change
+    change_table :professional_learning_partners do |t|
+      t.remove_index column: :name
+      t.index :name, unique: true
```

Comment:
Yeah, you're right. And I just confirmed on the source sheet that some PLPs have multiple entries, one for each contact.